internal/cmd/billing/command_cost_list.go

```diff
@@ -0,0 +1,114 @@
+package billing
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+type commandList struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type costOut struct {
+	StartDate           string `human:"Start Date" serialized:"start_date"`
+	EndDate             string `human:"End Date" serialized:"end_date"`
+	Granularity         string `human:"Granularity" serialized:"granularity"`
+	LineType            string `human:"Line Type" serialized:"line_type"`
+	Product             string `human:"Product" serialized:"product"`
+	ResourceId          string `human:"Resource ID" serialized:"resource_id"`
+	ResourceDisplayName string `human:"Resource Display Name" serialized:"resource_display_name"`
+	EnvironmentId       string `human:"Environment ID" serialized:"environment_id"`
+	NetworkAccessType   string `human:"Network Access Type" serialized:"network_access_type"`
+	Price               string `human:"Price" serialized:"price"`
+	Unit                string `human:"Unit" serialized:"unit"`
```

Comment:
We can combine these in the human output (should save us a column of space), and keep them separate in the serialized output.